packages/injective-protobuf/scripts/protoexport.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+echo "Exporting gogo proto files"
+#echo "Injective"
+#buf export https://github.com/InjectiveLabs/sdk-go.git --output ./proto/injective
```

Comment:
why commented out?

Reply:
It doesn't work. This requires adopting the gogo format / schema AFAIK.
That, or creating a repository in the BSR, and upload it there.

Reply:
Hmm, but this doesn't export all the required files. Working on it.

Reply:
Works now. Please take a look at the latest commits.